src/Routes.js

```diff
@@ -21,6 +21,11 @@ const InstanceDetailsPage = lazy(() =>
     /* webpackChunkName: "InstancesPage" */ './routes/InstanceDetailsPage/InstanceDetailsPage'
   )
 );
+const OverviewPage = lazy(() =>
+  import(
+    /* webpackChunkName: "OverviewPage" */ './routes/OverviewPage/OverviewPage'
```

Comment:
nit: Leading comment looks weird; can it be on the line above?

Reply:
Sadly, I think it's intended to be like this based on the starter template code I used